packages/core/src/components/Tabs/Tab/Tab.styled.tsx

```diff
@@ -37,9 +116,73 @@ export const Button = styled('button').attrs(({ theme: { tabs } }) => ({ tabs })
         cursor: not-allowed;
     }
 
-    ${SvgIcon} {
-        margin-bottom: 5px;
+    ${({ theme, active, tabSize, hasIcon }) => hasIcon && tabSize !== 'S' && svgStyles(theme, active, tabSize)}
```

Comment:
We could write like this ---> `${({ tabSize, hasIcon }) => hasIcon && tabSize !== 'S' && svgStyles}` styled-components automatically pass the props to tabs component.